geometry_msgs/Pose ee_pose
float32[] joint_angles

float32 pose_position_weight
float32 pose_orientation_weight
float32[] joint_angles_weights
float32 perlin_weight       # probably should be used as a switch
float32 collision_weight    # probably should be used as a switch